Source/WebKit/WPEPlatform/wpe/wayland/WPEDisplayWayland.cpp

```diff
@@ -219,32 +220,34 @@ const struct wl_registry_listener registryListener = {
         auto* display = WPE_DISPLAY_WAYLAND(data);
         auto* priv = display->priv;
 
-        if (!std::strcmp(interface, "wl_compositor"))
+        auto interfaceSpan = unsafeSpan(interface);
+
+        if (!equalSpans(interfaceSpan, "wl_compositor"_span))
```

Comment:
I would use `StringView::fromLatin1` and comparing against `ASCIILiteral` literals:

```suggestion
        const auto interfaceName = StringView::fromLatin1(interface);

        if (interfacename == "wl_compositor"_s)
```

I find using the overloaded `==` operator in this case clearer that adding the `equalSpans()` calls around.